### PR TITLE
[DE] Retranslate `Gameplay/Kiai_time`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Kiai_time/de.md [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
+*Für Regelungen zur Kiai-Zeit, siehe: [Ranking-Kriterien](/wiki/Ranking_Criteria)*
 
-![Sternenfontänen erscheinen, wenn der Kiai Time beginnt.](img/Kiai_Time_test.jpg "Sternenfontänen erscheinen, wenn der Kiai Time beginnt.")
+**Kiai-Zeit** (oder *Kiai*) ist ein Begriff, der für eine "spezielle" Timing-Sektion genutzt wird, um einen Teil einer [Beatmap](/wiki/Beatmap) besser zu betonen. Spieler können erkennen, ob eine Beatmap eine Kiai-Timing-Sektion hat, wenn sie zusätzliche visuelle Effekte im Hintergrund sehen (z. B. aufleuchtende Beats, fliegende Sterne und Fontänen).
 
-![In der oberen linken Ecke erscheint der Kiai Time, wenn er aktiv ist.](img/Kiai_Time_edit.jpg "In der oberen linken Ecke erscheint der Kiai Time, wenn er aktiv ist.")
+Während Kiai aktiv ist, leuchten alle [Hit-Objekte](/wiki/Hit_object) im Rhythmus des [Haupttakts](/wiki/Client/Beatmap_editor/Timing) auf und Sterne explodieren in der Nähe des Cursors, sobald ein [Hit-Circle](/wiki/Hit_object/Hit_circle) getroffen wurde. Anderweitig fallen sie vom Cursor herunter, wenn ein [Slider](/wiki/Hit_object/Slider) oder [Spinner](/wiki/Hit_object/Spinner) betätigt wurde.
```
Comment (The-Last-Cookie) [inline wiki/Gameplay/Kiai_time/de.md:15]:
```suggestion
Während Kiai aktiv ist, leuchten alle [Hit-Objekte](/wiki/Hit_object) im Rhythmus des [Haupttakts](/wiki/Client/Beatmap_editor/Timing) auf und Sterne explodieren in der Nähe des Cursors, sobald ein [Hit-Circle](/wiki/Hit_object/Hit_circle) getroffen wurde. Anderweitig fallen sie vom Cursor herunter, wenn ein [Slider](/wiki/Hit_object/Slider) geschoben oder ein [Spinner](/wiki/Hit_object/Spinner) gedreht wurde.
```

File: wiki/Gameplay/Kiai_time/de.md [hunk 2]
```diff
@@ -1,24 +1,23 @@
...
-3. **Benutzen Sie keine weiße Combofarben** (255,255,255); dadurch wird der Spieler geblendet und dessen Augen stark beansprucht.
-4. Es ist normal, dass eine Kiai-Sektion im Vergleich zum Rest der Map schwieriger ist. Siehe Definiton vom Kiai Time.
-5. **Die Kiai Time-Sektionen in [Gast Schwierigkeitsstufen](/wiki/Beatmap/Guest_difficulty) können sich von den Schwierigkeitsstufen des Beatmaperstellers unterscheiden**. Dies ist des öfteren der Fall, da nicht jeder denkt, dass die eine Stelle am besten für den Kiai geeignet ist; nichts desto trotz sollte sie jedoch passend zur Map definiert werden.
-6. Colourhaxing ist dazu da, um dem Kiai durch den Set aus verschiedenen Combofarben einen gewissen Touch zu geben.
+Auf [osu!taiko](/wiki/Game_mode/osu!taiko)-Beatmaps geben Kiai-Sektionen 20 % mehr [Score](/wiki/Gameplay/Score).
```
Comment (The-Last-Cookie) [inline wiki/Gameplay/Kiai_time/de.md:23]:
```suggestion
Auf [osu!taiko](/wiki/Game_mode/osu!taiko)-Beatmaps geben Kiai-Sektionen 20 % mehr [Punkte](/wiki/Gameplay/Score).
```

**Review by The-Last-Cookie — COMMENTED** [review]
Suggestions from my side

File: wiki/Gameplay/Kiai_time/de.md (hunk 1 above)
Comment (FamousPig) [reply wiki/Gameplay/Kiai_time/de.md:15]: Could also do `Slider gehalten`.

File: wiki/Gameplay/Kiai_time/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Gameplay/Kiai_time/de.md:23]: Differentiating between `Punkte` and `Score` is not easy. Therefore, I will use `Score` to reduce ambiguity.

File: wiki/Gameplay/Kiai_time/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Gameplay/Kiai_time/de.md:15]: Editing in the other suggestion, due to the two being in the same line.

**Review by FamousPig — APPROVED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**